job_bundles/Using_Files_In_Jobs.md

```diff
@@ -0,0 +1,1161 @@
+  --required-file-system-location-names-to-add FSComm FS2
+```
+
+Note that if a queue has any required filesystem locations, then that queue cannot be associated with a
+[service-managed fleet](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/smf-manage.html) because
+that fleet has no way to mount your shared filesystems.
+
+A queue's configuration also includes a list of allowed storage profiles that applies to jobs submitted to
+and fleets associated with that queue. Only storage profiles that define filesystem locations for all of the required filesystem
+locations of that queue are allowed in the queue's list of allowed storage profiles. 
+
+Submitting a job with a storage profile other than one in the list of allowed storage profiles for a queue will fail. A job with
+no storage profile can always be submitted to a queue.
+The workstation configurations labeled `WSAll` and `WS1` both have the required filesystem locations (`FSComm` and `FS1`) for queue `Q1`
+and need to be allowed to submit jobs to the queue. Similarly, workstation configurations `WSAll` and `WS2` meet the requirements for 
+queue `Q2` and need to be allowed to submit jobs to that queue. So, update both queue's configurations to allow jobs to be submitted
+with these storage profiles:
+
+```bash
+# Change the value of WSALL_ID to the identifier of the WSALL storage profile
+WSALL_ID=sp-00112233445566778899aabbccddeeff
+# Change the value of WS1 to the identifier of the WS1 storage profile
+WS1_ID=sp-00112233445566778899aabbccddeeff
+# Change the value of WS2 to the identifier of the WS2 storage profile
+WS2_ID=sp-00112233445566778899aabbccddeeff
+
+aws deadline update-queue --farm-id $FARM_ID --queue-id $QUEUE1_ID \
+  --allowed-storage-profile-ids-to-add $WSALL_ID $WS1_ID
+
+aws deadline update-queue --farm-id $FARM_ID --queue-id $QUEUE2_ID \
+  --allowed-storage-profile-ids-to-add $WSALL_ID $WS2_ID
+```
+
+If you were to try to add the `WS2` storage profile to the list of allowed storage profiles for queue `Q1` then it would fail:
+
+```bash
+$ aws deadline update-queue --farm-id $FARM_ID --queue-id $QUEUE1_ID \
+  --allowed-storage-profile-ids-to-add $WS2_ID
+
+An error occurred (ValidationException) when calling the UpdateQueue operation: Storage profile id: sp-00112233445566778899aabbccddeeff does not have required file system location: FS1
+```
+
+This is because the `WS2` storage profile does not contain a definition for the filesystem location named `FS1` that queue `Q1` requires.
+
+Associating a fleet that is configured with a storage profile that is not in the queue's list of allowed storage profiles will also fail. For example:
+
+```bash
+$ aws deadline create-queue-fleet-association --farm-id $FARM_ID \
+   --fleet-id $FLEET_ID \
+   --queue-id $QUEUE1_ID
+
+An error occurred (ValidationException) when calling the CreateQueueFleetAssociation operation: Mismatch between storage profile ids.
```

Comment:
I really like this example showing errors, very illustrative and hopefully helps out people following along!

Reply:
Thanks!